webpack/ForemanTasks/Components/TaskActions/UnlockModals.js

```diff
@@ -28,7 +28,7 @@ export const ForceUnlockModal = ({ onClick, id, selectedRowsLen }) => (
     title={__('Force Unlock')}
     body={sprintf(
       __(
-        `Resources for %s task(s) will be unlocked and will not prevent other tasks from being run. As the task might be still running, it should be avoided to use this unless you are really sure the task got stuck.`
+        `Resources for %s task(s) will be unlocked and will not prevent other tasks from being run. As the task(s) might be still running, it should be avoided to use this unless you are really sure the task(s) got stuck.`
```

Comment:
The button which leads here says "force cancel", maybe not confusing people with unlocking would be a good idea?

Reply:
This would mean that force unlock button would open force cancel modal in task details, should we rename it in task details?

Reply:
Yeah, it would be good to be consistent